bundle-size/webhooks.js

```diff
@@ -108,48 +110,31 @@ exports.installGitHubWebhooks = (app, db, githubUtils) => {
     const approverTeams = check.approving_teams
       ? check.approving_teams.split(',')
       : [];
-    if (approverTeams.length) {
-      // TODO(#617, danielrozenberg): use the result of `isApprover` and
-      // `isSuperApprover` instead of the legacy logic below.
-      const isApprover = (
-        await githubUtils.getTeamMembers(approverTeams)
-      ).includes(approver);
-      context.log(
-        `Approving user ${approver} of pull request ${pullRequestId}`,
-        isApprover ? 'is' : 'is NOT',
-        'a member of',
-        approverTeams
-      );
-    }
-
-    // TODO(#617, danielrozenberg): remove the legacy logic below.
-    if (!(await githubUtils.isBundleSizeApproverLegacy(approver))) {
-      return;
-    }
-
+    const isApprover = (
```

Comment:
`isSuperApprover || (...)` to potentially save API calls and such

Reply:
Done